data/tools/wmlindent

```diff
@@ -299,7 +299,16 @@ def convertor(linefilter, arglist, exclude):
                 print("wmlindent: no WML file found, exiting", file=sys.stderr)
                 sys.exit(1)
 
+def sigint_handler(signal, frame):
+    """This function defines what happens when the SIGINT signal is encountered by pressing ctrl-c during runtime. 
+    When ctrl-c is pressed, a one-line message is displayed and Python exits with Status 0, which refers to successful termination.
+    This overrides Python's default behavior of displaying a traceback when ctrl-c is pressed.
+    """
+    print ('Aborted by pressing ctrl-c')
+    sys.exit(0)
```

Comment:
Exiting with 0 seems very questionable.

Reply:
What is the Exit Status Number to use?

Reply:
At least something that is not 0. Ideally the process should kill itself and not just exit so that a shell like bash will handle the case of pressing ctrl-c on a terminal correctly. Further reading: https://www.cons.org/cracauer/sigint.html

Reply:
I dunno about Python, but in C at least there's macros you're supposed to use for the argument to `exit()`, rather than the raw numbers

Reply:
This has nothing to do with using signal numbers or names. The point is that terminating (probably in the middle of a run) on a signal should not count as a success (i.e. exit status 0). The exit status should indicate failure and ideally it should also indicate that the termination is because of the signal which cannot be done at all purely with the exit status, hence the process killing itself.

Reply:
I used 0 because I thought it counted as a success since it was what the user wanted. I can make this exit status 1.

Reply:
There is nothing fancy about calling an external command from an interactive shell or a script. That is pretty much the main thing you do. wmlindent for example is called from a CI script on every commit. If someone now runs the CI script themselves and wants to abort with ctrl-c when wmlindent happens to be running then nothing will abort except wmlindent. They'll have to press ctrl-c again and if more commands behave this way again and again...

Exit status 0 meaning success and anything else failure is the same on Windows. It's a pretty universal convention, not just for UNIX.

Reply:
How do I run the CI script? I want to compare behavior of aborting the CI script before and after this PR.